include/deal.II/grid/reference_cell.h

```diff
@@ -189,6 +189,12 @@ class ReferenceCell
    * @pre The template argument `dim` must equal the dimension
    *   of the space in which the reference cell you are querying
    *   lives (i.e., it must equal the result of get_dimension()).
+   *
+   * For ReferenceCells::Vertex, the reference cell is a zero-dimensional point
+   * in a zero-dimensional space: i.e., asking about the value of a polynomial
+   * doesn't make sense since there cannot be any independent variables. To
+   * enable dimension-independent programming we define the zero-dimensional
+   * value to be 1.
```

Comment:
That's clearly the right choice because these functions are tensor products of one-dimensional polynomials and the neutral value for an empty product is 1.